source/blog/2016-03-11-release-notes-6-34.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Finagle 6.34 Release Notes
+published: true
+post_author:
+  display_name: Vladimir Kostyukov
+  twitter: vkostyukov
+tags: Releases, Finagle, Util, Ostrich, Scrooge, TwitterServer
+---
+
+This release we've been focusing on improving both overall stability of Finagle components and
+debugging features. We also reverted a patch that cased [duplicate tag error][tagerr] that looks as
+follows.
+
+```
+com.twitter.finagle.mux.ServerError: Duplicate tag 6
+```
+
+With that said, we highly recommend upgrading from 6.33 to 6.34.
+
+### Highlights
+
+* Finagle now includes downstream/upstream addresses in its exceptions (see [RB 797082][rb797082])
```

Comment:
oh you did! hm, maybe instead of using RB XXX we can use the first 8 bytes of the sha for the link?